swf/src/read.rs

```diff
@@ -2589,7 +2589,7 @@ pub fn read_compression_type<R: Read>(mut input: R) -> Result<Compression> {
         b"FWS" => Compression::None,
         b"CWS" => Compression::Zlib,
         b"ZWS" => Compression::Lzma,
-        _ => return Err(Error::invalid_data("Invalid SWF")),
+        _ => return Err(Error::invalid_data(format!("Invalid SWF: {:?}", signature))),
```

Comment:
Seems like this isn't very helpful to print the signature? It would be gibberish for any non-SWF file.